lib/friendly_shipping/services/ship_engine/serialize_rate_estimate.rb

```diff
@@ -0,0 +1,32 @@
+module FriendlyShipping
+  module Services
+    class ShipEngine
+      class SerializeRateEstimate
```

Comment:
Maybe call this `SerializeRateEstimateRequest` or `SerializeRateEstimateBody`? Otherwise, it looks like it might be a PORO for a rate estimate that's returned by Ship Engine that we happen to be serializing for some reason.